config/demo/gas.toml

```diff
@@ -11,3 +11,9 @@ claim = 121995
 open = 143116
 fill = 76068 
 claim = 59953
+
+[gas.flows.eip3009_escrow]
+# Gas units captured by scripts/e2e/estimate_gas_eip3009_escrow.sh on local anvil
+open = 130254
+fill = 77298 
+claim = 21000
```

Comment:
This seems off, let's get more realistic estimates from local anvil simulation.
Also don't forget that these files are generated from `deployments.sh` and `setup_testnet.sh` so adding the changes here won't be permanent. You'll need to update it in the relevant generation scripts.